dashboard/src/shared/RpcError.ts

```diff
@@ -0,0 +1,24 @@
+export class RpcError extends Error {
+  // Following https://github.com/grpc/grpc-go/blob/master/internal/status/status.go#L140
+  static RPC_ERROR_REGEX = /rpc error: code = (\w+) desc = (.*)/;
```

Comment:
Until we can have GRPC code to properly throw a dedicated error object, we can only guess if it is an rpc error by checking the error message string.

Reply:
You should be able to check the grpc error code? At least, the gRPC dashboard code *should* be throwing an error object with a `.code` property. That's what we're using here already: https://github.com/kubeapps/kubeapps/blob/master/dashboard/src/shared/Auth.ts#L71-L88

Reply:
Thanks, will definitely take a look. It would be simpler.